app/src/main/java/com/github/se/icebreakrr/ui/profile/ProfileEdit.kt

```diff
@@ -103,6 +103,7 @@ fun ProfileEditingScreen(
 
   fun updateProfile() {
     profilesViewModel.updateProfile(getCopyOfProfile())
+    profilesViewModel.getSelfProfile()
   }
```

Comment:
As you prefer, but during my manual testing, I noticed that editing the tags in our profile does not update the SelfProfile, which prevents the sort by common tags from updating correctly (since it relies on SelfProfile that hasn’t been updated). The only way to get the sorting to work properly again is to restart the application, which then updates SelfProfile from Firebase.

If I remove this feature here, it will have to be added at some point anyway.

Reply:
I saw that bug too and fixed it in my part. You can safely remove it now and I'll ensure that the behavior is correct before merging my PR